core/domain/feedback_services_test.py

```diff
@@ -41,7 +41,8 @@ def test_feedback_ids(self):
         # The rest of the thread id should not have any full stops.
         self.assertNotIn('.', thread_id[len(EXP_ID) + 1:])
 
-        messages = feedback_services.get_messages(threadlist[0]['thread_id'])
+        messages = feedback_services.get_messages(
+            EXP_ID, threadlist[0]['thread_id'].split('.')[1])
```

Comment:
Ditto re using some kind of common method to get the thread_id from the 'full thread id'